node/src/components/gossiper/tests.rs

```diff
@@ -384,6 +394,7 @@ impl reactor::Reactor for Reactor {
                 self.contract_runtime
                     .handle_event(effect_builder, rng, *event),
             ),
+            Event::Chainspec(_request) => todo!(),
```

Comment:
Should this be `unreachable!` rather than `todo!` or did you mean to add something more here?

Reply:
Changed to `unreachable!` - this test reactor does not use chain spec requests